db/seeds.rb

```diff
@@ -1,7 +1,100 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+class Seed
+  def self.start(num_requesters, num_professionals, num_jobs)
+    seed = Seed.new
+    seed.create_skills
+    seed.create_requester(num_requesters)
+    seed.create_professional(num_professionals)
+    seed.create_jobs(num_jobs)
+  end
+
+  def create_skills
+    Skill.create(name: "Plumbing")
+    Skill.create(name: "Carpentry")
+    Skill.create(name: "Tailoring")
+    Skill.create(name: "Web Development")
+    Skill.create(name: "Blacksmithing")
+    Skill.create(name: "Electrician")
+    Skill.create(name: "General Contractor")
+    Skill.create(name: "Yoga")
+    Skill.create(name: "Mechanic")
+    Skill.create(name: "Animal Services")
+  end
+
+  def create_requester(num)
+    num.times do |i|
+      email = Faker::Internet.email
+      while User.find_by(email: email) do
```

Comment:
This is a good way to make sure it generates an email that passes the uniqueness validation.